examples/configurations/api/test.py

```diff
@@ -0,0 +1,10 @@
+import os
```

Comment:
maybe full config
I would but None in api_key
no specifying of max_tokens and temperature